internal/backend/policy/opapolicy/opa_test.go

```diff
@@ -0,0 +1,75 @@
+package opapolicy
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+var (
+	policyContent = `
+-- default/policy.rego --
+package policy
+
+import rego.v1
+
+passthrough := input.passthrough
+`
+
+	testfs = kittehs.Must1(kittehs.TxtarStringToFS(policyContent))
+)
+
+func TestPassthrough(t *testing.T) {
```

Comment:
It's unclear what this new test actually does. Document this better in the file and PR description.

Reply:
done